pkg/msr/msr_linux_amd64.go

```diff
@@ -274,13 +274,9 @@ func (m MSR) TestAndSet(c CPUs, clearMask uint64, setMask uint64) []error {
 // Locked verifies that for all MSRVal's for the CPU vendor, the MSRs are locked.
 // TODO: this is another Intel-specific function at present.
 func Locked() error {
-	vendor, err := cpuid.CPUManufacturerID()
-	if err != nil {
-		return err
-	}
-
+	vendor := cpuid.VendorIdentificatorString
 	// TODO: support more than Intel. Use the vendor id to look up msrs.
-	if vendor != cpuid.ManufacturerIDIntel {
+	if vendor != "GenuineIntel" {
```

Comment:
maybe it's better to have a constant for "GenuineIntel"?

Reply:
This was a constant that I newly introduced in the in-tree cpuid package. Since it is only really used in the msr command and there just string literal `"GenuineIntel"` is checked I think it is not necessary to abstract this.